ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
If no value is given, then Pulp Smash will assume that Pulp is the absolute latest development version. Do you think that's a sane default? We could do something with an interactive fact as you mention, though I don't know if either elyezer or I know how to do that.